packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -651,8 +652,15 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         return IdentifierWhitelistInterface(finder.getImplementationAddress(OracleInterfaces.IdentifierWhitelist));
     }
 
-    // Stamps the ancillary data blob with the optimistic oracle tag denoting what contract requested it.
     function _stampAncillaryData(bytes memory ancillaryData, address requester) internal pure returns (bytes memory) {
-        return abi.encodePacked(ancillaryData, "OptimisticOracle", requester);
+        // Since this contract will be the one to formally submit DVM price requests, its useful for voters to know who
+        // the original requester was.
+        bytes memory prefix;
+        if (ancillaryData.length > 0) {
+            prefix = ",optimisticOracleRequester:";
+        } else {
+            prefix = "optimisticOracleRequester:";
```

Comment:
Just to reduce gas, do you think we should shorten this to `ooRequester`?

Reply:
sure